Sources/ApolloCodegenLib/Frontend/JavaScript/src/index.ts

```diff
@@ -31,7 +33,37 @@ export {
 } from "graphql";
 export { GraphQLSchemaValidationError } from "./utilities/graphql";
 
-export function loadSchemaFromIntrospectionResult(
+export function loadSchemaFromSources(sources: Source[]): GraphQLSchema {
+  var introspectionJSONResult: Source | undefined
+
+  var documents = new Array<DocumentNode>()
+  for (const source of sources) {
+    if (source.name.endsWith(".json")) {
+      if (!introspectionJSONResult) {
+        introspectionJSONResult = source        
+      } else {
+        throw new Error(`Schema search paths can only include one JSON schema definition.
```

Comment:
Where does this constraint come from? Is graphql.js unable to merge multiple JSON sources? IMO extensions make sense in SDL but I wonder if there is an edge case where the schema is broken up into multiple JSON files; that may not even be possible though.

Either way I think this constraint should make it's way up into the header doc for `schemaSearchPaths`.

Reply:
The precondition just states the required extension but doesn't mention there can only be one. "schema files" could be interpreted to mean more than one.

> Precondition: JSON format schema files must have the file extension ".json".

Reply:
Oh youre right! I'll add that.